tests/Controller/EnMarche/AdherentControllerTest.php

```diff
@@ -40,7 +41,8 @@
  */
 class AdherentControllerTest extends WebTestCase
 {
-    use ControllerTestTrait;
+    use ControllerTestTrait,
```

Comment:
In this project we don't use the multiple short `use` syntax, can you write please this folowing declaration ?

```php
use ControllerTestTrait;
use MailTestCaseTrait;
```

Reply:
yes, thanks, fixed 👍